_security/authentication-backends/ldap.md

```diff
@@ -60,8 +60,21 @@ We provide a fully functional example that can help you understand how to use an
 
 To enable LDAP authentication and authorization, add the following lines to `config/opensearch-security/config.yml`:
 
+The internal user database authentication should also be enabled as OpenSearch Dashboards connects to OpenSearch using `kibanaserver` internal user
```

Comment:
```suggestion
The internal user database authentication should also be enabled because OpenSearch Dashboards connects to OpenSearch using the `kibanaserver` internal user.
```